onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -7,6 +7,9 @@
 #include "core/platform/ort_mutex.h"
 #include "core/platform/threadpool.h"
 
+#define TREEENSEMBLE_BATCHSIZE 256
+#define TREEENSEMBLE_MAXSIZE 8589934592
```

Comment:
constexpr?

Reply:
not used any more, sorry for leaving them